lib/buffer.js

```diff
@@ -993,62 +1014,64 @@ function checkOffset(offset, ext, length) {
 }
 
 
-Buffer.prototype.readUIntLE = function(offset, byteLength, noAssert) {
-  offset = offset >>> 0;
-  byteLength = byteLength >>> 0;
-  if (!noAssert)
-    checkOffset(offset, byteLength, this.length);
+Buffer.prototype.readUIntLE =
+  function readUIntLE(offset, byteLength, noAssert) {
+    offset = offset >>> 0;
+    byteLength = byteLength >>> 0;
+    if (!noAssert)
+      checkOffset(offset, byteLength, this.length);
 
-  var val = this[offset];
-  var mul = 1;
-  var i = 0;
-  while (++i < byteLength && (mul *= 0x100))
-    val += this[offset + i] * mul;
+    var val = this[offset];
```

Comment:
If we're churning let's get some butter: replace with for(...;...;...) to make it more readable. Performance should be on par